config/swagger.yaml

```diff
@@ -236,6 +236,27 @@ definitions:
       balance:
         type: integer
         format: int64
+  CoinbaseTx:
+    type: object
+    properties:
+      pubkey:
+        type: string
+      nonce:
+        type: integer 
+  SignedTx:
+    type: object
+    properties:
+      type:
+        type: string
+      data:
+        type: object
+        schema:
+          oneOf:
+            - $ref: '#/definitions/CoinbaseTx'
+      signatures:
+        type: array
+        items:
+          type: string
```

Comment:
I believe this type will need to be detailed further in order to be useful. I understand it shall contain both the signing pubkey and signature, in order for the client to use those to verify the signature on the transaction.

I see in backlog card "Verify signature of the payer on the transaction".

@velzevur If you understand enabling the HTTP client to verify the signature of the transaction is outside scope of GH-45, could you please add card in backlog?

Reply:
This type follows the current implementation of signatures:
https://github.com/aeternity/epoch/blob/master/apps/aecore/include/txs.hrl#L3
I agree with you that in order to validate a signature - a public key is needed so these should go as a pair. When the structure of #signed_tx.signatures changes - the corresponding section of .yaml should be changed, too.